src/components/pages/AdminAddCourses/AdminAddCourses.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import '../../../styles/index.less';
+import { Input, Form } from 'antd';
+
+export default function AdminAddCoursesForm(props) {
+  const [formValues, setFormValues] = useState({
+    subject: '',
+    description: '',
+    prereq: '',
+  });
+  const token = JSON.parse(localStorage.getItem('okta-token-storage'));
+  const config = {
+    headers: { Authorization: `Bearer ${token.idToken.value}` },
+  };
+  const body = {
+    ...formValues,
+    subject: formValues.subject,
+    description: formValues.description,
+    prereq: formValues.prereq,
+  };
+
+  function handleSubmit(e) {
+    axios
+      .post(`https://coder-heroes-api.herokuapp.com/course-types`, body, config)
```

Comment:
The endpoint will be changed to use env.variables rather than the hardcoded url.